x2b_default_config.toml

```diff
@@ -6,7 +6,6 @@ cpus-per-task = 2
 job-name = "xnat2bids"
 
 [xnat2bids-args]
-session = "XNAT_E00080"
+sessions = ["XNAT_E00080"]
```

Comment:
I mentioned this on another PR, but I'm learning towards not having a session specified in the default config and instead only have defaults that are likely not going to change for users